enterprise/static/enterprise/admin/manage_learners.js

```diff
@@ -1,22 +1,80 @@
-$(
-    function () {
-        $("table.learners-table .delete-cell input[type='button']").click(function () {
-            var $row = $(this).parents("tr");
-            var email = $row.data("email");
-            var message = interpolate(gettext("Are you sure you want to unlink %(email)s?"), {"email": email}, true);
-            if (confirm(message)) {
-                $.ajax({
-                    url: "?unlink_email=" + email,
-                    method: "delete",
-                    beforeSend: function (xhr) {
-                        xhr.setRequestHeader("X-CSRFToken", $.cookie("csrftoken"));
-                    }
-                }).success(function () {
-                    $row.remove();
-                }).error(function (xhr, errmsg, err) {
-                    alert(xhr.responseText);
-                })
+function disableMode() {
+    $("#id_course_mode").each(function (index, dropdown) {
+        dropdown.options.length = 0;
+        dropdown.options[0] = new Option(gettext('Enter a valid course ID'), '');
+        dropdown.disabled = true;
+    });
+}
+
+function fillModeDropdown(data) {
```

Comment:
can you add description that represent what this function `fillModeDropdown` is actually doing.